schema.graphql

```diff
@@ -0,0 +1,198 @@
+type Announcement {
+  announcementId: String!
+  company: String!
+  id: ID!
+  locationId: String!
+  message: String!
+  people: String!
+  publishOn: String!
+}
+
+type Event {
+  createdAt: String!
+  externalId: String!
+  id: ID!
+  source: EventSource!
+}
+
+type EventCollection {
+  all: [Event!]!
+  count: EventCount!
+}
+
+type EventCount {
+  githubCommit: Int!
+  githubPull: Int!
+  slackMessage: Int!
+}
+
+enum EventSource {
+  # Github commit
+  githubCommit
+
+  # Github pull request
+  githubPull
+
+  # Slack message
+  slackMessage
+}
+
+type Location {
+  bathroomCode: String!
+  cityName: String!
+  dayAnnouncements: Announcement!
+  id: ID!
+  isPrimary: Boolean!
+  latitude: Float!
+  longitude: Float!
+  timezone: String!
+  weather: Weather!
+  wifiName: String!
+  wifiPassword: String!
+}
+
+type Mutation {
+  testField: String!
+}
+
+type Query {
+  # MojoTech announcements
+  announcements(locationCityName: String, today: Boolean): [Announcement!]!
+
+  # MojoTech slack/github events
+  events(after: String, type: EventSource): EventCollection!
+
+  # MojoTech office locations
+  locations: [Location!]!
+
+  # Location running the app
+  primaryLocation: Location!
+
+  # Weather response from Dark Sky
+  weather(latitude: Float!, longitude: Float!): Weather!
+}
+
+type Subscription {
+  # An announcement was published
+  announcementPublished: Announcement!
+
+  # An event was published to the API
+  eventPublished: Event!
+
+  # Latest weather data retrieved
+  weatherPublished(latitude: Float!, longitude: Float!): Weather!
+}
+
+scalar UnixDateTime
+
+type Weather {
```

Comment:
These types being added seem unrelated to the commit was this file generated wholesale at some point?

Reply:
This file is automatically generated, not sure if I should still modify it?

Reply:
discussed offline. I think it would be cleaner to generate this file without the changes in this PR and make a new commit. then in this commit only include the changes that would be added to this file. This isn't the end of the world though.